stock_picking_group_by_partner_by_carrier_by_date/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking: group by partner and carrier and scheduled date",
+    "Summary": """Delivery orders will be matched by date too.
+    """,
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "category": "Warehouse Management",
+    "depends": ["stock_picking_group_by_partner_by_carrier"],
```

Comment:
A remark which may or may not apply here. `stock_picking_group_by_partner_by_carrier` adds a hook to customize the domain used in `_assign_picking`, which forces us to depend on `stock_picking_group_by_partner_by_carrier` if we want to customize the domain. Which was maybe necessary in your case? Anyway, not in my case :p, so at some point, it would be good to extract the hook in a base module, which is extended in `stock_picking_group_by_partner_by_carrier`, maybe in `stock_picking_group_by_scheduled_date` (?) and so on.

Reply:
I'll add a TODO.

Reply:
done